R_program/Main.R

```diff
@@ -470,22 +491,20 @@ sites<-c(notassigned)
 source(query.file)
 
 # these variables are no longer needed
-remove(climate.conditions)
 remove(query.file)
 remove(database)
+rm(split)
 
 ############################### End Weather Query Code ################################
 
 ############################### Weather Assembly Code #################################
 
-#This script assembles the necessary weather data that was extracted during the weather query step
-site<-c(site)
 #Set output directories
 weather.dir<-source.dir
 setwd(weather.dir)
 #Create a new folder called StepWat.Weather.Markov.Test in which to put the weather files and markov files
-dir.create("StepWat.Weather.Markov.Test", showWarnings = FALSE)
-assembly_output<-paste(source.dir,"StepWat.Weather.Markov.Test/",sep="")
+dir.create("MarkovWeatherFiles", showWarnings = FALSE)
+assembly_output<-paste(source.dir,"MarkovWeatherFiles/",sep="")
```

Comment:
Update comment here to replace `StepWat.Weather.Markov.Test `with `MarkovWeatherFiles`